src/main/java/frc/robot/commands/DefaultAimCommand.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.AimingConstants.AimState;
+import frc.robot.constants.AimingConstants;
+import frc.robot.constants.JoystickConstants;
+import frc.robot.subsystems.AimingSubsystem;
+import frc.robot.subsystems.Input;
+
+public class DefaultAimCommand extends Command {
+  private final AimingSubsystem mAimingSubsystem;
+
+  // Use the Arm from last year as inspiration
+  public DefaultAimCommand(AimingSubsystem aimingSubsystem) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    mAimingSubsystem = aimingSubsystem;
+    addRequirements(aimingSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    AimState state = AimState.STOW;
```

Comment:
With this, every cycle it resets to Stow unless you hold the button. Instead of making AimState.stow the default, don't change the desired Setpoint until a button is pressed.